src/components/CategoryFilter.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CategoryFilter = ({ filter, handleChangeFilter }) => {
+  const [selectedFilter, setSelectedFilter] = React.useState(filter.filter);
+  const categories = [
+    'All',
+    'Action',
+    'Biography',
+    'History',
+    'Horror',
+    'Kids',
+    'Learning',
+    'Sci-Fi',
+  ];
+
+  return (
+    <>
+      {categories.map((category) => (
+        <button className="category-selector" type="button" key={category} onClick={() => { setSelectedFilter(category); handleChangeFilter(category); }}>
+          <input key={category} type="radio" name="checkbox" value={category} checked={category === selectedFilter} onChange={() => {}} />
+          {category}
+        </button>
```

Comment:
- [ ] It's a better approach to use a Select tag here. That way we will have a value for each category. So when the category is selected in the Select tag, the value of the filter is updated.